sktime/annotation/clasp.py

```diff
@@ -281,7 +281,7 @@ def _predict_points(self, X):
             Series containing the indexes of the changepoints in X.
         """
         self.found_cps, self.profiles, self.scores = self._run_clasp(X)
-        return pd.Series(self.found_cps)
+        return pd.Series(X.index[self.found_cps].tolist())
```

Comment:
oh, I see, there is a shift by one.

The prize question is now, which of the two options is the one which is API compliant with the end point API definition? That is a question of encoding.

It may well be that it has not been recorded properly in the API specification, in which case it is a gap tha needs to be closed.

Reply:
Thanks for spotting this, @ShivamJ07. I have reverted this change in 37cc4a0.